src/sentry/consumers/__init__.py

```diff
@@ -378,6 +380,14 @@ def get_stream_processor(
     validate_schema: bool = False,
     group_instance_id: str | None = None,
 ) -> StreamProcessor:
+    from arroyo.backends.kafka.configuration import build_kafka_consumer_configuration
+    from arroyo.backends.kafka.consumer import KafkaConsumer
+    from arroyo.commit import ONCE_PER_SECOND
+    from arroyo.types import Topic as ArroyoTopic
+    from django.conf import settings
+
+    from sentry.utils import kafka_config
```

Comment:
a lot of these imports can be removed entirely -- or moved to the top of the file (some are already there)

Reply:
moved up the arroyo imports